### ISSUE TITLE
Missing columns in "genome_info.tsv" output

### ISSUE THREAD

**Issue by etd530** [MrOlm/inStrain#158]
Dear Matt,

When running inStrain for several BAM files mapped against the same set of genomes, I noticed that some columns in the `genome_info.tsv` output files were present in some cases, but not in others. Specifically, the columns are `linked_SNV_count`, `SNV_distance_mean`, `r2_mean` and `d_prime_mean`. Could this be a bug, or am I missing something?

The exact command was:
```
inStrain profile $file ../../wolbachia.genomes.dereplicated_ani99_maf90_concat.fna --stb ../../contigNames_all_dereplicated_ani99_maf90.tsv -o ${prefix}.drep_ani99_maf90.instrain -p 15 --database_mode
```

Where `$file` corresponds to a different BAM file each time.

Many thanks,

Eric

**Comment by etd530** [comment]
Hi Matt,
Thanks for explaining, now it makes sense :)
If I may, I would suggest that in future versions the program always outputs all columns even if one is completely empty, which would make it easier to compare output for multiple files.
Thanks again!
Eric

**Comment by MrOlm** [comment]
Fixed in v1.8.0
